pkg/cmd/secret/shared/shared.go

```diff
@@ -0,0 +1,7 @@
+package shared
+
+const (
+	VisAll      = "all"
+	VisPrivate  = "private"
+	VisSelected = "selected"
```

Comment:
Minor: when doing enums like this, would it make sense to declare a parent type as well? e.g.

```go
type Visibility string

const (
  VisibilityAll Visibility = "all"
  VisibilityPrivat Visibility = "private"
)
```